invenio_rdm_records/fixtures/tasks.py

```diff
@@ -44,11 +44,13 @@ def create_vocabulary_record(service_str, data):
 
 
 @shared_task
-def create_demo_record(user_id, data, publish=True):
+def create_demo_record(user_id_or_identity, data, publish=True):
```

Comment:
Since this is a Celery task, I'm not sure if `Identity` serializes properly... IIRC it's a simple namedtuple in the end, but it's always a hustle with Celery task parameters that need to be primitive/raw values.

Reply:
@fenekku did you have a chance to test if the identity serializes? I can merge it either way if @slint agrees

Reply:
I think we're good, since the only usage I see that calls `create_demo_record.delay(...)`, uses a user ID (which is serializable)

Reply:
I'll test Identity serialization out this morning.

Reply:
Ugh it's indeed not. I'll pass the system_user_id instead and convert in the creation function.